### ISSUE TITLE
Add view so people could see all hacks by theme

### ISSUE THREAD

**Issue by ligi** [berlin-hack-and-tell/berlinhackandtell.rocks#297]
just got a nice idea from a friend: it would be good if you could see all hacks of a certain theme (Musical, Hardware, Libraries, Apps, ..)
Currently this is quite hard as you need to go through all episodes to see them. Maybe also a view to see all hack-of-the-month's. 
For this we would also need a new data field like "tags" or "type"/"theme"

**Comment by lucasrangit** [comment]
Nice idea. I think adding tags/categories to pages is already possible. That would at least narrow it down to episodes that have talks that apply.

I'm not sure I understand what you mean by viewing all the hack of the months. Do you mean generating a page say with a list of all the hacks of the month only?

**Comment by ligi** [comment]
> I'm not sure I understand what you mean by viewing all the hack of the months. Do you mean generating a page say with a list of all the hacks of the month only?

yes - kinda. 
Think more like a page that has all hacks and then you can add filters e.g. for tags or hack-of-the-month

**Comment by vog** [comment]
Dear all,

I'd be very happy if we could start with a page that simply has all hacks.

That alone would already be a huge improvement, and in combination
with Ctrl+F would solve almost all needs.

Any further improvements could be added later on, in a second and third step.


Best regards,
Volker


ligi wrote:
> > I'm not sure I understand what you mean by viewing all the hack of the months. Do you mean generating a page say with a list of all the hacks of the month only?
> 
> yes - kinda. 
> Think more like a page that has all hacks and then you can add filters e.g. for tags or hack-of-the-month
> 
> -- 
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/berlin-hack-and-tell/berlinhackandtell.rocks/issues/297#issuecomment-932270551

-- 
Volker Diels-Grabsch
----<<<((()))>>>----

**Comment by ligi** [comment]
yes - makes sense - created #298 as a parent/dependency of this one
